project/MimaExcludes.scala

```diff
@@ -88,7 +88,12 @@ object MimaExcludes {
     // SPARK-47706: Bump json4s from 3.7.0-M11 to 4.0.7
     ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.sql.expressions.MutableAggregationBuffer.jsonValue"),
     ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.sql.types.DataType#JSortedObject.unapplySeq"),
-    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.mllib.tree.model.TreeEnsembleModel#SaveLoadV1_0.readMetadata")
+    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.mllib.tree.model.TreeEnsembleModel#SaveLoadV1_0.readMetadata"),
+    // SPARK-47814: Move `WriteInputFormatTestDataGenerator.scala` from `main` to `test`
```

Comment:
This seems to outdated. Could you match this with the PR title, @panbingkun ?

Reply:
Done